Compilers/Windows/gcc64.ey

```diff
@@ -1,30 +0,0 @@
-%e-yaml
```

Comment:
Is this file actually completely broken?

Reply:
Since we are going with mingw-w64 now it works just like Linux. You only need a ``gcc32.ey`` to force a 32 bit build, the default one will do a 64 bit build if available.